KIF Tests/AutocorrectTests_ViewTestActor.m

```diff
@@ -0,0 +1,60 @@
+//
+//  AutocorrectTests_ViewTestActor.m
+//  KIF Tests
+//
+//  Created by Harley Cooper on 2/7/18.
+//
+
+#import <KIF/KIF.h>
+
+@interface AutocorrectTests_ViewTestActor : KIFTestCase
+@end
+
+
+@implementation AutocorrectTests_ViewTestActor
+
++ (void)setUp
+{
+    [KIFTestActor setEnableAutocorrect:YES];
+    [KIFTestActor setEnableSmartQuotes:YES];
+    [KIFTestActor setEnableSmartDashes:YES];
+    [super setUp];
```

Comment:
It is generally encouraged to call `[super setUp]` first. I know this doesn't work with the current paradigm though. Putting a comment elsewhere with a suggestion.